tools/qc/fastq_quality_trimmer/.shed.yml

```diff
@@ -1,11 +1,10 @@
-name: fastx_toolkit
+name: fastq_quality_trimmer
 owner: ebi-gxa
 description: "Suite of FASTQ/A preprocessing tools"
 long_description: "The FASTX-Toolkit is a collection of command line tools for Short-Reads FASTA/FASTQ files preprocessing."
```

Comment:
if changing all to fastq, please do so in this line as well (there a FASTX part).

Reply:
Done in 5d98b3f51e6ebfb08204d909cd0dc6b4144b2a17